technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -28,8 +28,8 @@
       <term name="issue" form="short">H.</term>
       <term name="translator" form="verb-short">deutsch von</term>
       <term name="presented at">gehalten auf</term>
-      <term name="no date" form="short">o.&#8239;J.</term>
-      <term name="et-al">u.&#8239;a.</term>
+      <term name="no date" form="short">o. J.</term>
+      <term name="et-al">u. a.</term>
```

Comment:
And unless you can easily restore the escape codes, I'll just merge this as is and re-escape these characters later.